<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qsave</class>
 <widget class="QWidget" name="Qsave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qsave</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>选择路径</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox_2">
     <item>
      <property name="text">
       <string>JPG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PNG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BMP</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>提交图片名称</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="3" column="1" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
